android/app/src/main/AndroidManifest.xml

```diff
@@ -1,47 +1,33 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.iiitvadodara.club_calendar">
-    <!-- io.flutter.app.FlutterApplication is an android.app.Application that
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.iiitvadodara.club_calendar">
+	<!-- io.flutter.app.FlutterApplication is an android.app.Application that
          calls FlutterMain.startInitialization(this); in its onCreate method.
          In most cases you can leave this as-is, but you if you want to provide
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
-    <application
-        android:name="io.flutter.app.FlutterApplication"
-        android:label="club_calendar"
-        android:icon="@mipmap/ic_launcher">
-        <activity
-            android:name=".MainActivity"
-            android:launchMode="singleTop"
-            android:theme="@style/LaunchTheme"
-            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
-            android:hardwareAccelerated="true"
-            android:windowSoftInputMode="adjustResize">
-            <!-- Specifies an Android theme to apply to this Activity as soon as
+	<application android:name="io.flutter.app.FlutterApplication" android:label="club_calendar" android:icon="@mipmap/ic_launcher">
+		<activity android:name=".MainActivity" android:launchMode="singleTop" android:theme="@style/LaunchTheme" android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode" android:hardwareAccelerated="true" android:windowSoftInputMode="adjustResize">
+			<!-- Specifies an Android theme to apply to this Activity as soon as
```

Comment:
The lines are too large, could you use the previous default indentation for this file?

Reply:
yeah sure